.gitignore

```diff
@@ -1,34 +1,5 @@
-# built application files
```

Comment:
Why delete all these? It's healthy to ignore these files.

Reply:
its already covered by the root .gitignore file, although if you'd like to keep it that's fine